indico/modules/attachments/templates/_display.html

```diff
@@ -37,12 +37,14 @@
     {% set folders = item.attached_items.folders if item else folders %}
 
     <div class="attachments-display-container toolbar">
-        <div class="folder folder-root {{ 'folder-root-empty' if not files }}">
+        {% if files %}
```

Comment:
I don't think it is needed. `for` loop will not render anything if `files` is an empty array.